File: ApiSuvesaPos/Negocio/Modelo/cxp/ordencompra/ordencompra.vb
Namespace Modelo
    Public Class ordencompra
        Public Property Orden As Integer
        Public Property Proveedor As Integer
        Public Property Nombre As String
        Public Property Fecha As Date
        Public Property contado As Boolean
        Public Property credito As Boolean
        Public Property diascredito As Decimal
        Public Property Plazo As Integer
        Public Property Descuento As Decimal
        Public Property Impuesto As Decimal
        Public Property Total As Decimal
        Public Property Observaciones As String
        Public Property Usuario As String
        Public Property NombreUsuario As String
        Public Property entregar As String
        Public Property Cod_Moneda As Integer
        Public Property SubTotalGravado As Decimal
        Public Property SubTotalExento As Decimal
        Public Property SubTotal As Decimal
        Public Property Anulado As Boolean
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/ordencompra/ordencompra.cs
using System;

namespace Modelo
{
    public class ordencompra
    {
        public int Orden { get; set; }
        public int Proveedor { get; set; }
        public string Nombre { get; set; }
        public DateTime Fecha { get; set; }
        public bool contado { get; set; }
        public bool credito { get; set; }
        public decimal diascredito { get; set; }
        public int Plazo { get; set; }
        public decimal Descuento { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Total { get; set; }
        public string Observaciones { get; set; }
        public string Usuario { get; set; }
        public string NombreUsuario { get; set; }
        public string entregar { get; set; }
        public int Cod_Moneda { get; set; }
        public decimal SubTotalGravado { get; set; }
        public decimal SubTotalExento { get; set; }
        public decimal SubTotal { get; set; }
        public bool Anulado { get; set; }
    }
}

